modules/core/src/test/groovy/messages/MessageSourceTest.groovy

```diff
@@ -29,31 +30,37 @@ import spock.lang.Specification
 
 import org.springframework.beans.factory.annotation.Autowired
 
+import static test_support.TestLocales.*
+
 @ContextConfiguration(classes = [CoreConfiguration, TestAddon1Configuration, TestAppConfiguration])
 @TestExecutionListeners(value = AppContextTestExecutionListener,
         mergeMode = TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS)
 class MessageSourceTest extends Specification {
 
-    static final LOC_EN = Locale.ENGLISH
-    static final Locale LOC_RU = Locale.forLanguageTag('ru')
 
     @Autowired
     MessageSource messageSource
 
     @Autowired
     Environment environment
 
+    @Unroll
```

Comment:
I faced some trouble when run a test with `@Unroll` in IntelliJ. Could you please double-check it?

Reply:
Ive not seen any issues until now. What problems are popping up? Do use the gradle or the junit Test runner?

Reply:
I figured it out: IJ doesn't execute the test if you select this particular test method only. The whole test class runs fine.
So I think it's not a big problem.